src/styles/main.scss

```
@@ -1,3 +1,240 @@
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border: 0 solid white;
+  border-radius: 8px;
+  transition: 0.2s;
+}
+
+.is-active {
+  color: #00acdc;
+}
+/* #endregion */
+
+/* #region main */
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  transition: transform 300ms;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+/* #endregion */
+
+/* #region cards */
+$color: #00acdc;
+$color1: #060b35;
+
+.card {
+  width: 166px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    color: #{$color1};
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    line-height: 140%;
+    margin-bottom: 16px;
+  }
+
+  &__button {
+    display: flex;
+    text-decoration: none;
+    text-transform: uppercase;
+    box-sizing: border-box;
+    color: #fff;
+    align-items: center;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    height: 40px;
+    border: 1px solid #{$color};
+    border-radius: 5px;
+    background: #{$color};
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: #{$color};
+  }
+
+  &__product-code {
+    color: #616070;
+    font-size: 10px;
+    margin-bottom: 17px;
+  }
+}
+
+.card:hover {
+  transform: scale(120%);
+}
+
+.card:hover .card__name {
+  color: #34568b;
+}
+
+.price {
+  margin-bottom: 16px;
+  color: #616070;
+  font-size: 12px;
+}
+
+.text {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.rewiew {
+  color: #{$color1};
+  font-size: 10px;
+}
+
+.number {
+  color: #{$color1};
+  text-align: right;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 112.5%;
+}
+
+.stars {
+  display: flex;
+  &__star {
+    box-sizing: border-box;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

```suggestion
  @for $starCount from 1 through 5 {
    .start--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
      background-image: url(../images/star-active.svg);
    }
```

You can use sass loop here